tools/quickstart/quickstart/configs/tlbc/docker-compose.yaml

```diff
@@ -6,7 +6,7 @@
 version: "2.4"
 services:
   home-node:
-    image: trustlines/tlbc:release
+    image: trustlines/tlbc-node:release
```

Comment:
This tag does not exist yet, but we need it for testing, so I think we should here just use :master for now.

Reply:
But this tag [doesn't exit either](https://hub.docker.com/r/trustlines/tlbc-node/tags). Or what do I miss? I though for the tests we still stick to laika. And tlbc will then contain already the validators of the auction. :thinking: 
Sry for the circumstances...

Reply:
The tag exists on tlbc-node-next